examples/model_selection/plot_precision_recall.py

```diff
@@ -54,11 +54,32 @@
 The relationship between recall and precision can be observed in the
 stairstep area of the plot - at the edges of these steps a small change
 in the threshold considerably reduces precision, with only a minor gain in
-recall. See the corner at recall = .59, precision = .8 for an example of this
-phenomenon.
+recall.
+
+**Average precision** summarizes such a plot as the weighted mean of precisions
+achieved at each threshold, with the increase in recall from the previous
+threshold used as the weight:
+
+:math:`\\text{AP} = \\sum_n (R_n - R_{n-1}) P_n`
+
+where :math:`P_n` and :math:`R_n` are the precision and recall at the
+:math:`n`th threshold. A pair :math:`(R_k, P_k)` is referred to as an
+*operating point*.
+
+In *interpolated* average precision, a set of desired recall values is
+specified and for each desired value, we average the best precision scores
+possible with a recall value at least equal to the target value.
+The most common choice is 'eleven point' interpolated precision, where the
+desired recall values are [0, 0.1, 0.2, ..., 1.0]. This is the metric used in
+`The PASCAL Visual Object Classes (VOC) Challenge <http://citeseerx.ist.psu.edu
+/viewdoc/download?doi=10.1.1.157.5766&rep=rep1&type=pdf>`_. In the example
+below, the eleven precision values are indicated with an arrow to pointing to
+the best precision possible while meeting or exceeding the desired recall.
+Note that it's possible that the same operating point might correspond to
```

Comment:
This is in contradiction to the definition of operating point above. Do you simply mean that precision may be identical for multiple recall values?